app/src/main/java/com/instinctools/sprint_1/activity/MainActivity.java

```diff
@@ -0,0 +1,72 @@
+package com.instinctools.sprint_1.activity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.design.widget.Snackbar;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+import android.widget.CheckBox;
+import android.widget.EditText;
+import android.widget.Spinner;
+
+import com.instinctools.sprint_1.R;
+
+public class MainActivity extends AppCompatActivity implements View.OnClickListener {
+
+    public static final int[] mDrawableCompanyImg = {
```

Comment:
Try to create custom enum, See example https://www.tutorialspoint.com/javaexamples/method_enum1.htm.